superset-frontend/src/middleware/loggerMiddleware.js

```diff
@@ -116,7 +116,11 @@ const loggerMiddleware = store => next => action => {
     };
   }
 
-  if (eventData.target_id && dashboardLayout.present) {
+  if (
+    eventData.target_id &&
+    dashboardLayout.present &&
+    dashboardLayout.present[eventData.target_id]
```

Comment:
```suggestion
    dashboardLayout.present?.[eventData.target_id]
```
Nice suggestion @simcha90 - adding as a code suggestion here so the commit is just a click away for @geido :)